src/Gateway/MolliePaymentGateway.php

```diff
@@ -1148,4 +1090,89 @@ public function get_transaction_url($order): string
 
         return parent::get_transaction_url($order);
     }
+
+    /**
+     * @param string $currency
+     * @param $billingCountry
+     * @param string $paymentLocale
+     * @param array $filters
+     * @param $order_total
+     * @return bool
+     */
+    protected function isAvailableForSubscriptions(
```

Comment:
If you go one step further and put this in a separate class, you can 
1. Test it separately and more thoroughly
2. Mock it away when testing the actual MolliePaymentGateway, decreasing the complexity of tests while making it easier to test specific environments.